### ISSUE TITLE
Normalize Incident reporting

### ISSUE THREAD

**Issue by thekaveman** [cal-itp/benefits#1451]
We have a [Bug Report issue template](https://github.com/cal-itp/benefits/blob/dev/.github/ISSUE_TEMPLATE/bug-report.md) that attempts to capture some necessary details and is used for most all bug reports today in Benefits. We do not have a corresponding form yet for Server.

We need a normalized way to capture _Incidents_ (with a capital "I") that relate to downtime and are covered by the [Benefits Incident Management Plan](https://docs.google.com/document/d/1qtev8qItPiTB4Tp9FQ87XsLtWZ4HlNXqoe9vF2VuGcY/edit#heading=h.i5blb1qnnx67) (with associated _Severity_ and SLAs).

With GitHub's new Issue Forms syntax, we could both simplify the reporting and ensure relevant details are required / captured.

## Acceptance Criteria

- [ ] There is a standard way to report an Incident with all relevant details
 
## Additional context

Suggested list of fields:

### Required

- [ ] Description
- [ ] (Estimated) incident severity (single choice): options from (and link to) [Incident Management Plan](https://docs.google.com/document/d/1qtev8qItPiTB4Tp9FQ87XsLtWZ4HlNXqoe9vF2VuGcY/edit#heading=h.i5blb1qnnx67)
- [ ] Platform/OS (multiple choice): MacOS, iOS, Windows, Android, other?
- [ ] Browser (multiple choice): Chrome (latest), Chrome (prior), Safari (latest), Safari (prior), Firefox (latest), Firefox (prior), Edge (latest), Edge (prior), other?

### Optional

- [ ] Reproduction steps (text)
- [ ] Additional context (text)
- [ ] Screenshots (text?)

**Comment by thekaveman** [comment]
@indexing any thoughts / suggestions you would add here?

**Comment by indexing** [comment]
@thekaveman Will we use this template to create issues for _all_ bugs in the benefits and eligibility-server repositories?

**Comment by thekaveman** [comment]
@indexing we use the template for most bugs today in Benefits, we don't have a template in Server yet. 

Perhaps you're saying that _all bugs are not Incidents_ with a capital "I", and that's a good point. Maybe this task is really to create a template/form for _Incident_ in addition to our normal bug report form? The idea is we want to ensure we capture the `severity` in a normal way.

It would still be possible to create a regular issue and assign it the `bug` (or any) label, I think we want to keep this behavior around.

**Comment by indexing** [comment]
@thekaveman Yep. This is the direction I was thinking and we will want to align the approach across repos. I have an audit of labels slated for the next phase of the Github project (planned for late July / early August). I propose we address this issue as part of that effort. 

As you note, we need a way to designate a bug as an "incident," as many bugs aren't severe enough to qualify. We may not want to address a low severity bug in the next release or we may defer it for months in favor of higher priority work if it's minor. Maybe a label in the right attribute to categorize an issue as an incident?

Alternately, we could denote an issue as an incident by expanding the values for the priority field to include severity priorities below incident. (This approach mixes business and incident priorities in a unified way.) 

Lastly, we could modify the Incident Management Plan by adding a fifth severity level that sets no time to resolution, but that doesn't feel right for an incident. 

So the solution could be a combo of technical adjustments in Github and clarity in the process. Are you okay deferring this analysis for a month? (Feel free to assign to me.)

**Comment by machikoyasuda** [comment]
This ticket is kinda related? Could be combined: https://github.com/cal-itp/benefits/issues/1315

**Comment by thekaveman** [comment]
> This ticket is kinda related? Could be combined: #1315

I think that one was more related to checks and/or user-facing notifications that services are offline, based on the [Slack thread](https://cal-itp.slack.com/archives/C037Y3UE71P/p1678738734617289) it links to:

> Regarding the current [Login.gov](http://login.gov/) identity proofing outage :point_up: and our recent [Admin interface](https://github.com/cal-itp/benefits/milestone/30) workshop/discussion, it might be worth it to add to the scope "the ability to turn on a notification/alert banner"

This ticket is more about, how do we capture and track Incident reports. Definitely related but I think distinct enough to keep the tickets separate.
